Fabric/src/main/java/at/petrak/hexcasting/fabric/FabricHexInitializer.kt

```diff
@@ -97,6 +100,8 @@ object FabricHexInitializer : ModInitializer {
 
         HexLootFunctions.registerSerializers(bind(Registry.LOOT_FUNCTION_TYPE))
 
+        AkashicTreeGrower.INSTANCE
```

Comment:
This is to just reify the class re: Java lazy initialization of statics? (Perhaps add a comment or something to that effect cause just seeing that sitting by itself looks like a typo without the context)

Reply:
Personally, I think using static initializers that need to be loaded at a given time should be turned into invoked functions

Reply:
That's fair.  This approach had the smallest necessary change, but it definitely isn't obvious what it's doing.

Yeah, this is just to handle Java's lazy-initialization of AkashicTreeGrower, since its ``static{}`` call eventually uses ``FeatureUtils.register()``, which (in Fabric/Quilt and maybe vanilla?) has an explicit ``IllegalStateException("Registry is already frozen (trying to add key "")`` blocking that case.  Since there's no references to AkashicTreeGrower other than OpEdifySapling, it doesn't get called until someone casts that spell, and then the server side gives up.

Forge does allows those registrations available later, so the static init works fine in Forge cases.  But it's Not Good to load the same registry key twice; depending on modloader it will either crash or give a bunch of annoying [warn] messages.  The simple fix is to add the same registration invoke call into Forge's initializer, and that's what I've done for this commit.  

If that verboseness isn't ideal, you could keep the ``GROWERS.add`` calls in ``OpEdifySapling``'s ``static{}`` and then have an empty (or meaningless) ``RegisterTreeGrowers()`` that solely exists to instantiate the static variables.  I've had bad luck in the past with compiler optimizations pulling that sorta thing out, and it's not much clearer, though.

Reply:
I'd be OK with it as it is with a comment! That's what they're there for after all ...

Reply:
yrsegal put together some pretty good options for ``init()`` functions, so the current commit uses them.  I can switch back and comment if you prefer that code style; it's really about what's going to be clearest for you all.